ios/NotificationService/NotificationService.swift

```diff
@@ -75,6 +79,44 @@ class NotificationService: UNNotificationServiceExtension {
     }
   }
   
+  private func sendInvalidNotificationIntent() {
+    guard let notification = bestAttemptContent else { return }
+    if #available(iOSApplicationExtension 15.0, *) {
+      os_log(OSLogType.default, "Mattermost Notifications: creating invalid intent")
+      
+      bestAttemptContent?.body = "We could not verify this notification with the server"
```

Comment:
This could be translated.

Also move out of the if statement and assign to the declared notification variable

Reply:
I added the translation, but couldn't make it work. Not sure if a problem of working locally (I ran the replace_assets lane and even though didn't seem to work), but since I am following the same pattern as in other places in the code, I don't think is a problem exactly on this app.